services/webdav/pkg/config/config.go

```diff
@@ -18,9 +18,8 @@ type Config struct {
 
 	HTTP HTTP `yaml:"http"`
 
-	OcisPublicURL   string `yaml:"ocis_public_url" env:"OCIS_URL;OCIS_PUBLIC_URL" desc:"URL, where oCIS is reachable for users."`
-	WebdavNamespace string `yaml:"webdav_namespace" env:"WEBDAV_WEBDAV_NAMESPACE" desc:"CS3 path layout to use when forwarding /webdav requests"`
-	RevaGateway     string `yaml:"reva_gateway" env:"REVA_GATEWAY" desc:"The CS3 gateway endpoint."`
-
-	Context context.Context `yaml:"-"`
+	OcisPublicURL   string          `yaml:"ocis_public_url" env:"OCIS_URL;OCIS_PUBLIC_URL" desc:"URL, where oCIS is reachable for users."`
+	WebdavNamespace string          `yaml:"webdav_namespace" env:"WEBDAV_WEBDAV_NAMESPACE" desc:"CS3 path layout to use when forwarding /webdav requests"`
```

Comment:
`CS3 path layout...` is not very descriptive...

Reply:
Yes I agree. But this really belongs to a separate PR. Would you mind reporting this as a separate issue.

Reply:
This is a description text, why not making it correct here?

Reply:
Mainly because it doesn't  fit into this PR. Also I don't know from the top of my head what it actually does. Which means I need to dig into the code for finding it out while I would prefer to currently focus on the TLS work. Hence my suggestion to open a new issue. So we won't loose it. (Hint: there is a "Reference new issue" next to every issue comment, which should make that very easy)